rpc-server.py

```diff
@@ -129,8 +138,8 @@ def run_rpc_server(port, no_token):
     print(app_port, file=sys.stdout, flush=True)
 
     # Redirect stdout and stderror to log files
-    sys.stdout = open('rpc-server.out', 'w')
-    sys.stderr = open('rpc-server.err', 'w')
+    # sys.stdout = open('rpc-server.out', 'w')
+    # sys.stderr = open('rpc-server.err', 'w')
```

Comment:
I think that for development is ok to comment this. But I'm thinking that on the future, we could add an argument for if the user wants it on a file or not.